apps/aecore/lib/aecore/chain/chain_state.ex

```diff
@@ -68,29 +72,59 @@ defmodule Aecore.Chain.ChainState do
 
   @spec calculate_total_tokens(map()) :: integer()
   def calculate_total_tokens(chain_state) do
-    chain_state |>
-      Enum.map(fn{_account, data} -> data.balance end) |>
-      Enum.sum()
+    chain_state
```

Comment:
maybe return a tuple of {total_tokens, total_unlocked_tokens, total_locked_tokens}